cfme/infrastructure/virtual_machines.py

```diff
@@ -222,15 +222,24 @@ def on_details(self, force=False, is_vm=True):
             else:
                 return False
 
-    def _find_quadicon(self, is_vm=True, do_not_navigate=False, mark=False, refresh=True):
+    def _find_quadicon(
+            self, is_vm=True, do_not_navigate=False, mark=False, refresh=True, any=False):
         """Find and return a quadicon belonging to a specific vm
 
+        Args:
+            any: Whether to look for it anywhere (root of the tree)
+
         Returns: :py:class:`cfme.web_ui.Quadicon` instance
         Raises: VmNotFound
         """
         quadicon = Quadicon(self.name, "vm")
         if not do_not_navigate:
-            if is_vm:
+            if any:
```

Comment:
Though I get it may cause issues, as we are unifying and standardizing, shouldn't we ensure these functions are identical across clouds and infra to ensure they can both work in the same way? There's a part of me that wonders if we can have a base class where find_quadicon just uses certain constants for the force navigates etc. I am 100% behind the unification, we have too much divergence and with some of the new stuff coming this is going to become more important.